src/index.html

```diff
@@ -7,8 +7,60 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#logo" class="logo">
+        <img src="./images/logo.png" alt="Logo">
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+
+          <li class="nav__item">
```

Comment:
```suggestion
    <header class="header">
      <a href="#logo" class="logo">
        <img src="./images/logo.png" alt="Logo">
      </a>

      <nav class="nav">
        <ul class="nav__list">

          <li class="nav__item">
```

No need to add an empty line between parent and child elements, add an empty line only between multiple sibling blocks of html